cmd/verify-pr/verify-pr.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	// "log"
+
+	"github.com/google/go-github/v49/github"
+	"github.com/konveyor/release-tools/action"
+	"github.com/konveyor/release-tools/pr"
+)
+
+func main() {
+
+	ghContext, err := action.VarsFromEnv()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Parse the event
+	event, err := func() (github.PullRequestEvent, error) {
```

Comment:
Is there a reason to do this in an anonymous function? you could add the defer inline right?

Reply:
No. Yes.

Parts copied without enough :brain: